cloudinary-image-management-and-manipulation-in-the-cloud-cdn/js/src/components/video.js

```diff
@@ -82,62 +89,57 @@ const TransformationsToggle = ( props ) => {
 	);
 };
 
-const cldFilterBlocksEdit = ( BlockEdit ) => {
+let ImageInspectorControls = ( props ) => {
+	const {setAttributes, media} = props;
+	const {InspectorControls} = wp.editor;
+	
+	if ( media && media.transformations ) {
+		setAttributes( {transformations: true} );
+	}
+
+	return (
+		<InspectorControls>
+			<TransformationsToggle {...props} />
+		</InspectorControls>
+	);
+};
+
+ImageInspectorControls = withSelect( ( select, ownProps ) => ( {
+  ...ownProps,
+  media: ownProps.attributes.id ? select( 'core' ).getMedia( ownProps.attributes.id ) : null
+} ))( ImageInspectorControls );
 
-	const EnhancedBlockEdit = function( props ) {
+const cldFilterBlocksEdit = ( BlockEdit ) => {
+	return ( props ) => {
 		const {name} = props;
+		const shouldDisplayInspector = 'core/image' === name || 'core/video' === name
 
-		let inspectorControls;
-		if ( 'core/image' === name || 'core/video' === name ) {
-			inspectorControls = cldImageInspectorControls( props );
-		}
 		return (
 			<>
-				{inspectorControls}
+				{shouldDisplayInspector ? <ImageInspectorControls {...props} /> : null}
```

Comment:
```suggestion
				{shouldDisplayInspector && <ImageInspectorControls {...props} />}
```

Reply:
I prefer the verbosity of the ternary operator 😄

Reply:
Haha